api/pkg/utils/config/testdata/opensds.conf

```diff
@@ -12,7 +12,7 @@ log_file = /var/log/opensds/osdsdock.log
 # Choose the type of dock resource, only support 'provisioner' and 'attacher'.
 dock_type = provisioner
 # Specify which backends should be enabled, sample,ceph,cinder,lvm and so on.
-enabled_backends = cinder
+enabled_backends = sample,ceph,cinder,lvm
```

Comment:
Didn't understand the purpose of these changes

Reply:
corresponding tests were failing with IndexOutOfBound Exceptino when "enabled_backend[1] , enabled_backend[2] were accessed.